pkg/data/egress_lists/egress_lists.go

```diff
@@ -91,12 +102,42 @@ func EgressListToString(egressListYamlStr string, variables map[string]string) (
 				tlsDisabledURLListStr += urlStr
 				continue
 			}
+
+			// ECR registry ID from OCM or CLI passed as slice, and converted to a joint string by comma
+			if strings.Contains(urlStr, ",") && strings.Contains(urlStr, "dkr.ecr") {
+				urlStr = ConvertECRRegistryURLs(urlStr)
+			}
+
 			urlListStr += urlStr
 		}
 	}
 	return urlListStr, tlsDisabledURLListStr, nil
 }
 
+// ConvertECRRegistryURLs separates the multiple ECR urls
```

Comment:
Assuming this function has the core logic to handle the input of the constructed ECR URL.  So, if we have 2 ECR IDs, the constructed ECR URL should be something like `"123456789,987654321.dkr.ecr.us-west-2.amazonaws.com"`  (As I can see the ECR ID is defined as 12 digits?)

I just created a test.go file to call your function `ConvertECRRegistryURLs`  passing this dummy URL string:
```
func main() {
	// Example input for multiple ECR registry IDs
	result := ConvertECRRegistryURLs("123456789012,210987654321.dkr.ecr.us-west-2.amazonaws.com")
	fmt.Println("Constructed URLs:", result)
}
```
Looks the output is concatenated as below, is that expected? 
```
go run test.go
Constructed URLs: 123456789012.dkr.ecr.us-west-2.amazonaws.com210987654321.dkr.ecr.us-west-2.amazonaws.com
```

Reply:
thanks for testing @xiaoyu74 
As the function is nested in the for loop within the `EgressListToString`, so it will have the prefix `https://`, suffix `:443` appended from the yaml template and and an extra space added between each url